components/core/tools/scripts/utils/build-with-docker.py

```
@@ -0,0 +1,143 @@
+import argparse
+import logging
+import os
+import pathlib
```

Flake8 (7.2.0)

[error] 'pathlib' imported but unused

(F401)